requirements.txt

```diff
@@ -13,3 +13,4 @@ jsonschema==2.5.1
 ansible==2.1.0
 coverage
 nose
+python-openstackclient==0.8.6
```

Comment:
@herlo : These are the requirements for testing, However nose comes by default in most of the machines. I added it for just in case if we could not find. Further  os_hot_stack ansible module is an implementation of python-openstackclient. That's why we need them in requirements.txt